tests/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM python:2.7.18
+
+RUN apt update \
+    && apt install -y protobuf-compiler
```

```suggestion
RUN apt-get update \
    && apt-get install -y protobuf-compiler=<you can fix a version here if needed for reproducibility>
    && rm -rf /var/lib/apt/lists/*
```
Using `apt-get` will remove this warning:

```
[2020-07-30T14:58:35.412Z] WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```

The last line (`rm -rf /var/lib/apt/lists/*`) removes caches and reduces the size of the image.